src/Poll.ts

```diff
@@ -83,8 +68,8 @@ export class Poll {
         // Since its databaseless the way we know if it is anonymous or multiple is by parsing the title
         this.multiple = this.checkIfMsgContains("(Multiple Answers)");
         this.anonymous = this.checkIfMsgContains("(Anonymous)");
-        // If there's no buttons then the poll is locked
-        this.isLocked = this.message[3].type === "divider";
+        //if the is a lock symbol right below the divider, the poll is locked
+        this.isLocked = (this.message.length-1 === this.getDividerId())? false : ((this.message[this.getDividerId()+1] as SectionBlock).text as MrkdwnElement).text === ":lock:";
```

Comment:
I'm all for ternaries but this one is very difficult to read and I would prefer if it was split into an if else as I cannot decipher this one and its very crammed.

Reply:
Fixed